python/cuml/feature_extraction/text.py

```diff
@@ -15,13 +15,15 @@
 import warnings
 
 from cudf import Series
+from sklearn.exceptions import NotFittedError
 from sklearn.feature_extraction.text import ENGLISH_STOP_WORDS
```

Comment:
We don’t have an explicit dependency on Scikit-learn and this will end up blowing up for users that have not explicitly installed it. I believe there are a few other places where this is being imported. We should remove them and perhaps use our own NotFittedError.

Reply:
Good point, I removed the dependency on Scikit-learn from this PR. I think the other places where we import `NotFittedError` should be handled in a separate PR. I created an issue for this: https://github.com/rapidsai/cuml/issues/2523.